devicon.json

```diff
@@ -3054,6 +3054,26 @@
             }
         ]
     },
+    {
+        "name": "hibernate",
+        "altnames": [],
+        "tags": [
+            "database",
+            "framework"
```

Comment:
Isn't this Java related, too?
```suggestion
            "framework",
            "java"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question